task1/css_form.css

```diff
@@ -0,0 +1,43 @@
+body {
+    width: 400px;
+    height: 410px;
+    margin: 200px 500px 200px 500px;
+    background-color: whitesmoke;
+    align-content: center;
+
+}
+#container{
+    border: 1px lightgray;
+    border-style: groove;
+    border-top-style:none;
+}
+legend {
+    height: 40px;
+    width: 400px;
+    background-color: yellowgreen;
+    font: bold 15px Arial, Helvetica, sans-serif;
+    color: white;
+    text-transform: uppercase;
+    box-sizing: border-box;
+    text-align:center;
+    line-height:2.5;
+
+}
+
+input {
```

Comment:
You may want to style elements through classes instead of doing it this way in order to get into the habit of doing it.